src/rgw/driver/sfs/object.cc

```diff
@@ -319,7 +319,7 @@ int SFSObject::set_obj_attrs(const DoutPrefixProvider *dpp,
                              Attrs *delattrs,
                              optional_yield y) {
   ceph_assert(objref);
-  auto meta = objref->meta;
+  auto &meta = objref->meta;
```

Comment:
these changes seem part of a different logical commit than the one described by this commit?

Reply:
done